src/screens/Resources/LoadScreen.js

```diff
@@ -0,0 +1,46 @@
+import firestore from '@react-native-firebase/firestore';
+import { useState, useCallback } from 'react';
+import { useFocusEffect } from 'react-navigation-hooks';
+import PropTypes from 'prop-types';
+import { collections, nav } from '../../constants';
+
+export default function useLoadScreen(navigation) {
+  const [loading, setLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  async function firestoreQuery(collection, id) {
+    return firestore()
+      .collection(collection)
+      .where('parent', '==', id)
+      .get();
+  }
+
+  async function loadScreen(id, title) {
+    setLoading(true);
+
+    try {
```

Comment:
we do that because otherwise it gets kinda messy and unclear bc i have to keep changing .then() statements. the async await generally makes it a bit clearer.

Reply:
Sounds good!